bin/bossplot

```diff
@@ -142,7 +175,8 @@ def main():
         return -1
 
     # Initialize the plot.
-    figure = plt.figure(figsize=(12,8))
+    figsize = [float(value) for value in args.figsize.split(',')]
+    figure = plt.figure(figsize=figsize, tight_layout=True)
```

Comment:
I did this to quiet a warning, see here:

https://github.com/matplotlib/matplotlib/issues/1852